src/middlewared/middlewared/plugins/account.py

```diff
@@ -88,6 +88,33 @@ class Config:
                 pass
         return user
 
+    @accepts(Ref('query-filters'), Ref('query-options'), Bool('dssearch', default=False))
```

Comment:
To not change .query behavior we would need to somehow use the same parameters, only filters and options, using `@filterable` here instead of accepts, meaning we would have to drop dssearch.